docs/Configuration.rst

```diff
@@ -4837,7 +4840,59 @@ database.
 -  The **EventCategoriesCache** size is approximately 24 bytes, so the
    maximum heap size equals to the cache size multiplied by 24 bytes.
    
-   
+
+Wallet caches
+~~~~~~~~~~~~~
+
+eXo Platform provides 2 wallet caches:
+
+-  :ref:`Wallet account cache <Wallet.accountCache>`
+
+-  :ref:`Wallet transaction cache <Wallet.transactionCache>`
+
+You can add new values of these caches in :ref:`exo.properties <Configuration.ConfigurationOverview>` file.
+
+.. code-block:: jproperties
+
+    exo.cache.wallet.account.MaxNodes=2000
+    exo.cache.wallet.account.TimeToLive=-1
+    exo.cache.wallet.transactions.MaxNodes=2000
+    exo.cache.wallet.transactions.TimeToLive=-1
+
+
+The specific configuration of wallet caches can be found in:
+
+-  ``$PLATFORM_TOMCAT_HOME/webapps/wallet-common.war!/WEB-INF/wallet/cache-configuration.xml``.
+
+.. _Wallet.transactionCache:   
+
+Wallet account cache
+-------------------------
+
+The **Wallet account cache** caches the Wallet objects.
+This object contains metadata information of a wallet, such as identity id, username, space pretty name, wallet enablement...
+When any user get access to their wallet or a space wallet, the cached wallet object will be retrieved from cache rather than the database.
+
+-  The cached **Wallet account** is invalidated when the user updates its metadata such as reimporting its private key.
+
+-  The **Wallet account cache** size should equals the number of wallets used in eXo Platform.
+
+-  Each Wallet object is approximately 400 bytes, so the maximum heap size equals to the cache size multiplied by 400 bytes.
+
+.. _Wallet.transactionCache:   
+
+Wallet Transactions cache
+-------------------------
+
+The **Wallet Transactions cache** caches the blockchain transaction metadata sent by a wallet. This object contains information such as transaction hash, username, space pretty name, transaction status...
+When any user get access to the list of their transactions or a space wallet transactions, the cached wallet transactions object will be retrieved from cache rather than the database.
+
+-  The cached **Wallet transaction** is invalidated when the users updates their metadata such as changing transaction status from *pending* to *success* or *failed*.
+
+-  The **Wallet transaction cache** size should equals the number of wallet accounts multiplied by the first transactions list page size (10 elements per page).
+
+-  Each Wallet transaction object is approximately 700 bytes, so the maximum heap size equals to the cache size multiplied by 700 bytes.
+
```

Comment:
Each Wallet transaction object take about 700 bytes in memory. So the maximum heap size occupied by the wallet transaction cache would be its size multiplied by 700 bytes

Reply:
done